sound/soc/sof/ops.h

```diff
@@ -476,6 +476,19 @@ snd_sof_ipc_pcm_params(struct snd_sof_dev *sdev,
 	return sof_ops(sdev)->ipc_pcm_params(sdev, substream, reply);
 }
 
+/* host side configuration of the pipeline's offset in stream mailbox area */
+static inline int
+snd_sof_set_pipeline_offset(struct snd_sof_dev *sdev,
```

Comment:
set_pipeline_mb_offset?

pipeline_offset is a weird concept in itself, I think we need a hint in the name that tells the reader what the offset applies to.

Reply:
I also wonder if it's not a stream offset rather. The notion of pipeline is a container really, used for scheduling purposes in the firmware, but it's not really visible from the host. The only thing that is known is a stream.

Reply:
Hmm, stream_offset is a bit too open, if we would define it as `stream_data_offset'?
The reason is that it is the offset of the data (pcm or compressed) for the given stream.

Reply:
I'll keep the `_set_`, without it it looks more like a variable for the first sight:
```c
.stream_data_offset = hda_stream_data_offset,
```
compared to
```c
.set_stream_data_offset = hda_set_stream_data_offset,
```

Reply:
stream_data_offset sounds good to me, good suggestion.